sdks/python/apache_beam/transforms/enrichment_handlers/bigtable.py

```diff
@@ -81,7 +81,7 @@ def __init__(
       table_id: str,
       row_key: str,
       row_filter: Optional[RowFilter] = CellsColumnLimitFilter(1),
-      app_profile_id: str = None,
+      app_profile_id: str = "",
```

Comment:
Looking at https://cloud.google.com/python/docs/reference/bigtable/latest/google.cloud.bigtable.table.Table I think None is actually the correct default

Reply:
yeah, added a `type: ignore[assignment]` to escape from mypy